docs/en/your-profile-and-preferences/customize-community-and-channel-notifications.md

```diff
@@ -0,0 +1,68 @@
+---
+id: 585
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customize community and channel notifications
+
+Any member of an existing community or channel can set up their notification settings in three different options: [`mute messages`](link), [`mention people`](link) and [`set up exceptions`](link). Using these options, you receive notifications only from those communities or channels that are relevant to you. 
+
+This article describes common examples.
+
+- [Mute all community notifications.](#mute-all-community-notifications)
+- [Mute community notifications except global mentions.](#mute-community-notifications-except-global-mentions)
+- [Set the same settings for all community messages types.](#set-the-same-settings-for-all-community-messages-types)
+
+## Mute all community notifications
+
+You can turn off your notifications for specific communities for a designated period. Communities appear greyed out when you mute their notifications.
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities**.
+    1. Long press the community you want to mute.
+    1. Tap :mobile-mute: **Mute**.
+    1. Select the length of time you'd like to mute your notifications.
+
+!!! tip
+	Communities appear greyed out :mobile-mute: **Mute**, when you mute their notifications.
+
+=== "Desktop"  
+
+    1. From the navigation sidebar, click :desktop-settings: **Settings**.
+    1. From the settings sidebar, click :desktop-communities: **Communities**.
+    1. Choose the community you want to mute, and click :desktop-notification: **Mute**.
+
+## Mute community notifications except global mentions
+
+If you want to avoid missing important messages, you can selectively mute some notifications, except global mentions.
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities**.
+    1. Long press the community you want to mute.
+    1. Tap :mobile-notifications-settings: **Notification settings**.
+    1. Select **Personal mentions** > **Notifications off**.
+    1. Select **All other messages** > **Notifications off**.
+    1. Verify **Global mentions** is set to **On**. If it is not, then select **Global mentions** > **Notifications on**.
```

Comment:
'Make sure **Global mentions** is set to **On**. If it's not, select **Global mentions** <...>'